versioned_docs/version-6.x/react-native-screens.md

```diff
@@ -8,16 +8,37 @@ Prior to `2.14.0`, all screens are essentially regular native `View` in each pla
 
 With the advent of `react-native-screens`, the native screen optimization is brought possible to React Navigation by bringing the native navigation component (`UIViewController` for iOS, and `FragmentActivity` for Android). By using `react-native-screens`, it is possible for each native platform to optimize the memory usage for screens that are under the view stack and also simplify the native node hierarchy. You can take a look at the comparison [here](https://twitter.com/janicduplessis/status/1039979591815897088?s=21) to see the performance gain.
 
+React Navigation 6 incorporates `react-native-screens` in version >=3.0.0 which means the optimizations are already enabled by default.
 ## Setup when you are using Expo
 
-By default expo already included `react-native-screens`, all you need to do is pasting the following snippet before your navigation stacks are rendered (typically in an `index.js` or `App.js` file):
+Expo SDK 41 and up comes with `react-native-screens` that works out-of-the-box without additional configuration. If you installed dependencies with `expo install` as stated in [Getting started](http://localhost:3000/docs/6.x/getting-started#installing-dependencies-into-an-expo-managed-project) section of documentation you are good to go!
 
-```js
-// Before rendering any navigation stack
-import { enableScreens } from 'react-native-screens';
-enableScreens();
+## Setup in normal react-native applications
+
+### Setup on iOS
+
+The installation of `react-native-screens` on iOS is supported by auto-linking, if you made sure the pods are installed with `pod install` in project directory, no other actions are necessary.
+
+### Setup on Android
+
+This change is required to avoid crashes related to View state being not persisted consistently across Activity restarts.
+
+You'll need to edit `MainActivity.java` which is located in `android/app/src/main/java/<your package name>/MainActivity.java` file.
+
+Add this method to the body of `MainActivity` class:
+```java
+@Override
+protected void onCreate(Bundle savedInstanceState) {
+  super.onCreate(null);
+}
```

Comment:
I think we need to move this to getting started docs.

Also not sure what to do about this page since screens is already enabled by default now. Maybe let's remove this page for now?

We can add a more generic performance optimization page later with more details about screens and other optimizations.

Reply:
Fixed in 83bedbd